contracts/src/CrossChainApplications/examples/NativeTokenBridge/tests/NativeTokenDestinationTests.t.sol

```diff
@@ -198,17 +217,20 @@ contract NativeTokenDestinationTest is NativeTokenBridgeTest {
         );
         assertEq(_DEFAULT_INITIAL_RESERVE_IMBALANCE, nativeTokenDestination.totalSupply());
 
+        // Receive a transfer that will fully collateralize the bridge and mint some tokens
         vm.expectEmit(true, true, true, true, address(nativeTokenDestination));
         emit CollateralAdded({
             amount: _DEFAULT_INITIAL_RESERVE_IMBALANCE - firstTransfer,
             remaining: 0
         });
+
+        vm.expectEmit(true, true, true, true, address(nativeTokenDestination));
         emit NativeTokensMinted(_DEFAULT_RECIPIENT, firstTransfer);
 
         vm.expectCall(
             NATIVE_MINTER_PRECOMPILE_ADDRESS,
             abi.encodeWithSelector(INativeMinter.mintNativeCoin.selector),
-            1
+            2
```

Comment:
can you put how `2` was computed instead of hardcoding it incase other values get changed.

Reply:
This is the number of calls we expect to the Native Minter precompile, so there's no variable to put here.

Reply:
oh that's ncie I didn't even know `expectCall` had the amount of times, and can be set to 0.

Reply:
Could we break this up to two `expectCalls` that specify the specific expected amount to be minted in each call in the call data?